### ISSUE TITLE
'Expected the file "Rewired_Core.dll"' error on Windows 10.

### ISSUE THREAD

**Issue by totalsource** [zeobviouslyfakeacc/ModLoaderInstaller#32]
I downloaded TLD v1.56 to try some old mods, but the Modloader throws the above error. It's basically the same issue as the Mac user has in one of the other threads, but then it can't be, since I am on Windows. Any ideas?

**Comment by totalsource** [comment]
Downloaded version appeared to be v1.56, but is really v1.8. Thanks for pointing it out. Will post update if and when it is solved.

**Comment by totalsource** [comment]
When downloading the old version, I used DepotDownloader instead of using the Steam Console. For some reason the Console grabbed the wrong game version. Using DepotDownloader to grab the old manifest instead of using the Steam Console does the trick. I followed Verulean's guide on how to get it done, linked here: youtube.com/watch?v=44HBxzC_RTg
When you get to the cmd command, copy/paste the following string to download v1.56 of TLD:
dotnet DepotDownloader.dll -app 305620 -depot 305621 -manifest 8350414681168775351 -username <USERNAME> -password <PASSWORD>

Game version is now v1.56 and ModLoader should work fine.
